apps/test/unit/utilsTest.js

```diff
@@ -801,4 +804,41 @@ describe('utils modules', () => {
       expect(result).to.equal('blah"');
     });
   });
+
+  describe('firehoseDataSize', () => {
+    const maxDataJSONBytes = 65500;
+    const maxDataStringBytes = 4095;
+
+    beforeEach(() => {
+      stub(logToCloud, 'logError');
+    });
+
+    afterEach(() => {
+      logToCloud.logError.restore();
+    });
+
+    it('checks json size to send newrelic error', () => {
+      const valid_record = {data_json: 'x'.repeat(maxDataJSONBytes - 1)};
+      validateFirehoseDataSize(valid_record);
+      assert(logToCloud.logError.notCalled);
+
+      const invalid_record = {data_json: 'x'.repeat(maxDataJSONBytes + 1)};
+      expect(() => {
+        validateFirehoseDataSize(invalid_record);
+      }).not.to.throw();
+      assert(logToCloud.logError.calledOnce);
+    });
```

Comment:
please update these tests to check the return value from `validateFirehoseDataSize`, here and below

Reply:
roger that!